web/pdf_viewer.js

```diff
@@ -2169,7 +2169,11 @@ class PDFViewer {
     for (const entry of entries) {
       if (entry.target === this.container) {
         this.#updateContainerHeightCss(
-          Math.floor(entry.borderBoxSize[0].blockSize)
+          Math.floor(
+            entry.borderBoxSize?.length > 0
+              ? entry.borderBoxSize[0].blockSize
+              : entry.contentRect.height
+          )
```

Comment:
Since we don't want useless code in the Firefox PDF Viewer (i.e. the primary reason for this library existing), please update this to something like the following to limit this code to only `legacy`-builds.
```suggestion
          // Safari doesn't support `borderBoxSize` until version 15.4.
          Math.floor(
            typeof PDFJSDev !== "undefined" &&
              !PDFJSDev.test("SKIP_BABEL") &&
              !entry.borderBoxSize?.length
              ? entry.contentRect.height
              : entry.borderBoxSize[0].blockSize
          )
```

Reply:
done

Reply:
Done. Please check.

Reply:
I think that commit message should contain a link to the MDN compatibility data, to help a future reader.

Reply:
It's ok.

Reply:
Please keep in mind that commit messages could (and in most cases should) be more than just a single line for readability. Hence please move the "see https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry/borderBoxSize#browser_compatibility" part to its own line.

Reply:
done